pkg/manager/impl/task_manager.go

```diff
@@ -120,6 +121,15 @@ func (t *TaskManager) CreateTask(
 			contextWithRuntimeMeta, common.RuntimeVersionKey, finalizedRequest.Spec.Template.Metadata.Runtime.Version)
 		t.metrics.Registered.Inc(contextWithRuntimeMeta)
 	}
+
+	if request.Spec.DescriptionEntity != nil {
+		err = createDescriptionEntity(ctx, t.db, request.Spec.DescriptionEntity, *request.Id)
+		if err != nil {
```

Comment:
if the error is already exists, that's okay right?

Reply:
removed it. Now, creating tasks, creating description entity, and updating description ID are done in one DB transaction.